contracts/CO30_Jonty_JackalsLoot.Woodland_ACR/description.ext

```diff
@@ -2,10 +2,14 @@
 #include "edit_me\description.ext"
 
 class CfgSounds {
-    class Speak01 {
-        name = "Speak01";
-        sound[] = {"intel.ogg", 5, 1};
+    class Officer {
+        name = "Officer";
         titles[] = {};
-        duration = 10;
```

Comment:
Why did the duration go from 10 to 6? It needs to be the duration of your audio file, which I know is much longer